src/compositions/__tests__/useLspFactory.spec.ts

```diff
@@ -1,30 +1,13 @@
 import { useLspFactory } from "@/compositions/useLspFactory";
+import { NETWORK_URL, PRIVATE_KEY } from "@/helpers/config";
+import { getState } from "@/stores";
+import { LSPFactory } from "@lukso/lsp-factory.js";
 
 jest.mock("@lukso/lsp-factory.js", () => ({
-  LSPFactory: jest.fn().mockImplementation(() => ({
-    ProxyDeployer: {
-      deployBaseContracts: () => ({
-        universalProfile: {
-          address: "0x1",
-        },
-        universalReceiverDelegate: {
-          address: "0x1",
-        },
-      }),
-    },
-    LSP3UniversalProfile: {
-      deploy: () => "success",
-    },
-  })),
+  LSPFactory: jest.fn(),
 }));
-jest.mock("@/services/provider.service", () => ({
-  getSigner: async () => ({
-    provider: {
-      getNetwork: jest.fn().mockImplementation(async () => ({
-        chainId: 22,
-      })),
-    },
-  }),
+jest.mock("@/stores", () => ({
+  getState: jest.fn().mockReturnValue(22),
```

Comment:
Let's keep mocking in minimum. You don't need to mock state, just use `setState` in test